### PR TITLE
Add a reload-plugin button for debugging

### LINKED ISSUE #115: Reloading of plugins
Add a button to the plugin manager to reload plugins.
Should only be available of debug is true (or add a new config option development or something)

### PR DESCRIPTION
**Pull request by nycex**
Using the remove-plugin image, so the image has to be changed. Fixed #115

### CODE REVIEW

File: engine/plugins/plugins/style.css [hunk 1]
```diff
@@ -117,6 +117,22 @@
...
   font-size: 0;
   transition: background-color ease .2s;
+  margin-bottom: 5px;
+  border: none;
+  min-width: 0;
+  min-height: 0;
+}
+
+.DI-plugins-button-reload {
+  background: url(data:image/svg+xml;base64,PHN2ZyBmaWxsPSIjRkZGRkZGIiBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxwYXRoIGQ9Ik0xOSA2LjQxTDE3LjU5IDUgMTIgMTAuNTkgNi40MSA1IDUgNi40MSAxMC41OSAxMiA1IDE3LjU5IDYuNDEgMTkgMTIgMTMuNDEgMTcuNTkgMTkgMTkgMTcuNTkgMTMuNDEgMTJ6Ii8+CiAgICA8cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+Cjwvc3ZnPg==)
```
Comment (cking) [inline engine/plugins/plugins/style.css]: choose an icon for example from [flaticon](https://www.flaticon.com/search?word=reload)
use a data uri converter to paste it

**Comment by nycex** [comment]
Done.

**Comment by cking** [comment]
can u attach a screenshot please?

**Comment by nycex** [comment]
Here: https://tinyimg.io/i/DFAfGhh.png

**Pull request merged**